app/src/androidTest/java/org/dhis2/usescases/searchte/entity/DisplayListFieldsUIModel.kt

```diff
@@ -0,0 +1,11 @@
+package org.dhis2.usescases.searchte.entity
+
+data class DisplayListFieldsUIModel(
+    val name: String,
+    val lastName: String,
+    val birthday: String,
+    val email: String,
+    val address: String,
+    val weight: String,
+    val height: String
```

Comment:
For now this is okay. If at some point we use a different program this attributes could change. If this happens in the future we could use Map<String, String> instead and make both value/key dynamic